src/Compilers/Core/Portable/MetadataReader/MetadataDecoder.cs

```diff
@@ -1101,29 +1101,8 @@ private void DecodeParameterOrThrow(ref BlobReader signatureReader, /*out*/ ref
             if (typeCode == SignatureTypeCode.ByReference)
             {
                 info.IsByRef = true;
-
-                // The spec says that custom modifiers must precede SignatureTypeCode.ByReference, but the managed C++
-                // compiler emits them in the reverse order.  In order to avoid breaking interop scenarios, we need to
-                // support decoding (and later emitting) such signatures.
```

Comment:
The spec is being amended to allow signatures like that, I also think that the comment is no longer valuable.